src/components/StorageOverview/StorageOverview.js

```diff
@@ -12,12 +12,20 @@ import OCSHealthConnected from './OCSHealth/Health';
 import { CapacityConnected } from './Capacity/Capacity';
 import EventsConnected from './Events/Events';
 import { UtilizationConnected } from './Utilization/Utilization';
+import { AlertsConnected } from './Alerts/Alerts';
 
 const MainCards = () => (
   <GridItem lg={6} md={12} sm={12}>
     <Grid>
       <GridItem span={12}>
-        <OCSHealthConnected />
+        <Grid className="kubevirt-cluster-overview__health-grid">
+          <GridItem span={12}>
+            <OCSHealthConnected />
+          </GridItem>
+          <GridItem span={12}>
+            <AlertsConnected className="kubevirt-cluster-overview__card--top-border" />
```

Comment:
can you please move this to `sass/Dashboard/dashboard.scss` if it is used in more places?

Reply:
ack